settings/src/components/totp.js

```diff
@@ -225,7 +225,10 @@ function SetupForm( { handleEnable, qrCodeUrl, secretKey, inputs, setInputs, err
 	}, [ error, inputs ] );
 
 	const handleComplete = useCallback( ( isComplete ) => setIsInputComplete( isComplete ), [] );
```

Comment:
Unrelated, but from my understanding there is no benefit to using `useCallback` with a `useState` setter function, as it is stable between renders. We can pass `setIsInputComplete` to the child component directly.

Reply:
I don't know React very well :) but that sounds good to me.